src/main/kotlin/Decorator/littlekai/base/Noodles.kt

```diff
@@ -1,5 +1,6 @@
 package Decorator.littlekai.base
 
-interface  Noodles {
-  fun calculateCost() : Double
+interface Noodles {
+  fun calculateCost(): Double
+  fun calculateTotalCost(): Double
```

Comment:
But If `Noodles` already has a method `calculateTotalCost()` then, every child that implements it modifies its behaviour. `Noodles` shouldn't have that method, it belongs to `IngredientDecorator` and the concrete decorators which will add that behaviour to Noodles.